Mf2/Parser.php

```diff
@@ -1578,7 +1578,7 @@ public function backcompat(DOMElement $el, $context = '', $isParentMf2 = false)
 					if ( $location->length ) {
 						foreach ( $location as $tempEl ) {
 							if ( !$this->hasRootMf2($tempEl) ) {
-								$this->addMfClasses($tempEl, 'h-card');
+								$this->addMfClasses($tempEl, 'p-location h-card');
 								$this->backcompat($tempEl, 'vcard');
```

Comment:
Needs some type of `if` in here somewhere.

If the location element does not have and `adr` or `vcard` class it needs to be handled as a plain `p-location` string. I _think_ `adr` and `vcard` are both parsed into an h-card, if I am mistaken those two cases also need separate handling.